ethpm/validation.py

```diff
@@ -18,11 +25,24 @@ def validate_package_name(pkg_name: str) -> None:
         raise UriNotSupportedError('{0} is not a valid package name.'.format(pkg_name))
 
 
+def validate_registry_uri(uri: str) -> None:
+    """
+    Raises exception if URI does not conform the the registry URI scheme.
```

Comment:
>Validate a registry uri. Raise exception if URI does not conform the registry URI scheme

Reply:
IDK - starting with "Validate a registry URI" seems slightly redundant to me - since it's almost word for word the name of the function. For you, does it help with clarity? I'm thinking ...
When the `validate_registry_uri` is called it will:
> Raise an exception if the URI does not conform to the registry URI scheme.

Reply:
Sounds good :+1: